changelog.md

```diff
@@ -1,5 +1,11 @@
 # chartmogul-ruby Change Log
 
+## Version 1.3.1 - 6 May 2019
+- Add #last method to Entries
+
+## Version 1.3.0 - 5 March 2019
+- Method #update_cancellation_dates added to allow users updating the cancellation dates on subscription. The previous one (#set_cancellation_dates) was not sending the update to Chartmogul API.
```

Comment:
I think is to allow users `to update` the cancellation

Reply:
I copied it from Releases and it's not a typo. It was "not sending the update" ;)